src/main/java/javax/validation/Configuration.java

```diff
@@ -254,6 +269,17 @@
 	ClockProvider getDefaultClockProvider();
 
 	/**
+	 * Returns the default {@link ValueExtractor} implementations as per the
+	 * specification.
+	 *
+	 * @return the default {@code ValueExtractor} implementations compliant
+	 * with the specification
+	 *
+	 * @since 2.0
+	 */
+	Set<ValueExtractor<?>> getDefaultValueExtractors();
```

Comment:
I'm not very convinced of the usefulness of this, added it though for the sake of consistency. I don't think an extractor will be useful in a delegation pattern (which is why there are the other getDefault... methods).